src/Sylius/Component/Core/Model/Order.php

```diff
@@ -318,6 +318,20 @@ public function getLastPayment($state = null)
     }
 
     /**
+     * @return bool
+     */
+    public function requiresShipping()
```

Comment:
That was an unfortunate name, cause there is already `ProductVariant::isShippingRequired()`. Let's be concise and make it the same, wdyt?